src/main/java/com/example/calender/entity/Office.java

```diff
@@ -0,0 +1,25 @@
+package com.example.calender.entity;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class Office {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "OFFICE_ID")
```

Comment:
Why use capital letters for field names? using lower case would be better as it would help in speed typing as MySQL is case-sensitive